pkgs/by-name/do/docbook2x/package.nix

```diff
@@ -16,8 +16,9 @@ stdenv.mkDerivation rec {
   # writes its output to stdout instead of creating a file.
   patches = [ ./db2x_texixml-to-stdout.patch ];
 
-  nativeBuildInputs = [ makeWrapper perlPackages.perl texinfo libxslt ];
-  buildInputs = [ groff libxml2 opensp libiconv iconv bash ]
+  strictDpes = true;
```

Comment:
`strictDeps = true;`

There is a typo there!

Reply:
Ooops, sorry, and thank you for spotting this. Fix: https://github.com/NixOS/nixpkgs/pull/400315